bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractBaseImpl.java

```diff
@@ -156,6 +164,10 @@ public Label getLabel() {
 
     @Override
     public @Nullable String getTooltip() {
+        if (xssApi != null) {
+            logger.info("[AF] XSS API has been used to encode tooltip HTML.");
```

Comment:
You need to handle the else case too, when xssAPI Is not available. Please get ride of info logs from here

Reply:
Else cases are already handled right below the if block since there's a return inside.